packages/replay-next/components/inspector/values/PrototypeRenderer.tsx

```diff
@@ -0,0 +1,40 @@
+import { Object as ProtocolObject, PauseId as ProtocolPauseId } from "@replayio/protocol";
+import { useContext } from "react";
+
+import Expandable from "replay-next/components/Expandable";
+import { objectCache } from "replay-next/src/suspense/ObjectPreviews";
+import { ReplayClientContext } from "shared/client/ReplayClientContext";
+
+import PropertiesRenderer, { addPathSegment } from "../PropertiesRenderer";
+import styles from "./shared.module.css";
+
+type PropertiesRendererProps = {
+  path: string | undefined;
+  object: ProtocolObject;
+  pauseId: ProtocolPauseId;
+};
+
+export default function PrototypeRenderer({ object, pauseId, path }: PropertiesRendererProps) {
```

Comment:
Can we move this into a different folder? It's not a "value" renderer like the other things in here. It's rendering prototype properties.